test/test_cases/run_1d_ekman_column.jl

```diff
@@ -81,8 +81,10 @@ function run_1d_ekman_column(
         # Generate simple simulation data for test
         simulation = Simulation(model, stepper, dt = dt, tspan = (0.0, dt))
         @unpack ρ, uv, w, ρθ = init_1d_ekman_column(FT, params)
-        set!(simulation, :base, ρ = ρ, uh = uv, w = w)
-        set!(simulation, :thermodynamics, ρθ = ρθ)
+        set!(model.base.ρ, ρ)
```

Comment:
@bischtob, do you know what this should be?

Reply:
Oh, maybe we need `set!(simulation.integrator.ρ)` or `set!(simulation.integrator.base.ρ)` etc.

Reply:
Yeah, that makes more sense.